src/modeliteration.js

```diff
@@ -336,18 +336,21 @@ define([
         if (Boolean(value && value.idsQuery) !== Boolean(previous && previous.idsQuery)) {
             var nodeID = mug.p.nodeID,
                 currentPath = mug.absolutePath,
+                hashPath = mug.hashtagPath,
                 oldParent = mug.parentMug,
-                oldPath;
+                oldPath, oldHash;
             if (value && value.idsQuery) {
                 oldPath = currentPath.replace(/\/item$/, "");
+                oldHash = hashPath.replace(/\/item$/, "");
             } else {
                 oldPath = currentPath + "/item";
+                oldHash = hashPath + "/item";
                 if (/\/@count$/.test(mug.p.repeat_count)) {
                     mug.p.repeat_count = "";
                 }
             }
             mug.form.vellum.handleMugRename(
-                mug.form, mug, nodeID, nodeID, currentPath, oldPath, oldParent);
+                mug.form, mug, nodeID, nodeID, hashPath, oldHash, oldParent);
```

Comment:
Looks like `oldPath` and `currentPath` are not used anymore. Rather than modifying the names here, would it make sense to change `mug.absolutePath` to `mug.hashtagPath` and not change this function in any other way?